source/features/show-names.tsx

```diff
@@ -24,10 +24,15 @@ async function dropExtraCopy(link: HTMLAnchorElement): Promise<void> {
 function appendName(element: HTMLAnchorElement, fullName: string): void {
 	// If it's a regular comment author, add it outside <strong> otherwise it's something like "User added some commits"
 	const {parentElement} = element;
-	const insertionPoint = parentElement!.tagName === 'STRONG' ? parentElement! : element;
+	const isStrong = parentElement?.tagName === 'STRONG';
```

Comment:
```suggestion
	const isStrong = parentElement!.tagName === 'STRONG';
```

This is never missing, we should always prefer `!.` because `?.` silences future bugs